### PR TITLE
[Windows] nvme hot extend issue fixed in ESXi 8.0U1

### PR DESCRIPTION
**Pull request by 123lzxm**
change the condition for the nvme 1.3 hot extend known issue

### CODE REVIEW

**Comment by 123lzxm** [comment]
```
Testbed information:
+------------------------------------------------------------------------------------------+
| Product | Version | Build    | Hostname or IP | Server Model                             |
+------------------------------------------------------------------------------------------+
| vCenter | 8.0.1   | 21560480 |   |                                          |
+------------------------------------------------------------------------------------------+
| ESXi    | 8.0.1   | 21495797 |  | VMware, Inc. VMware7,1                   |
|         |         |          |                | Intel(R) Xeon(R) Gold 6230 CPU @ 2.10GHz |
+------------------------------------------------------------------------------------------+


VM information:
+-------------------------------------------------------------------------------------------------+
| Name                      | test_vm                                                             |
+-------------------------------------------------------------------------------------------------+
| Guest OS Distribution     | Microsoft Windows 11 Enterprise Insider Preview 10.0.25346.0 64-bit |
+-------------------------------------------------------------------------------------------------+
| IP                        |                                                      |
+-------------------------------------------------------------------------------------------------+
| Hardware Version          | vmx-20                                                              |
+-------------------------------------------------------------------------------------------------+
| VMTools Version           |                                                                     |
+-------------------------------------------------------------------------------------------------+
| Config Guest Id           | windows11_64Guest                                                   |
+-------------------------------------------------------------------------------------------------+
| GuestInfo Guest Id        |                                                                     |
+-------------------------------------------------------------------------------------------------+
| GuestInfo Guest Full Name |                                                                     |
+-------------------------------------------------------------------------------------------------+
| GuestInfo Guest Family    |                                                                     |
+-------------------------------------------------------------------------------------------------+
| GuestInfo Detailed Data   |                                                                     |
+-------------------------------------------------------------------------------------------------+


Test Results (Total: 1, Passed: 1, Elapsed Time: 00:08:31)
+-------------------------------------------------------+
| ID | Name                        | Status | Exec Time |
+-------------------------------------------------------+
|  1 | nvme_disk_hot_extend_spec13 | Passed | 00:08:06  |
+-------------------------------------------------------+
```

File: windows/vhba_hot_add_remove/hot_extend_disk_test.yml [hunk 1]
```diff
@@ -41,7 +41,7 @@
     - name: "Known issue - workaround of hot extend NVMe disk size"
       ansible.builtin.debug:
         msg:
-          - "Hot extended NVMe disk size is not recognized in Windows guest when NVMe Spec v1.3 is emulated. Ignore this known issue on ESXi version <= 8.0 GA build 20513097."
+          - "Hot extended NVMe disk size is not recognized in Windows guest when NVMe Spec v1.3 is emulated. Ignore this known issue on ESXi version < 8.0 U1 GA build 21495797."
```
Comment (keirazhang) [inline windows/vhba_hot_add_remove/hot_extend_disk_test.yml:44]: Replace `8.0 U1 GA` with `8.0 Update 1`

Comment (123lzxm) [reply windows/vhba_hot_add_remove/hot_extend_disk_test.yml]: Fixed. Thanks.

**Review by keirazhang — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**